frontend/src/api/backendClient.ts

```diff
@@ -1,3 +1,12 @@
+export interface User {
+  userName: string
+  firstName: string
+  lastName: string
+  email: string
+  totalPointsEarned: number
+  aboutMe: string
```

Comment:
It looks like none of these fields are nullable here but in the `ProfileResponse` that's being mapped onto this object some of the fields are nullable - I think and they should match

Reply:
All fixed :)